linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_all_shadowed_sha512/oval/shared.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
+  <definition class="compliance" id="accounts_password_all_shadowed_sha512" version="1">
+    <metadata>
```

Comment:
please use the [oval_metadata macro](https://complianceascode.readthedocs.io/en/latest/jinja_macros/oval.html?highlight=oval_metadata#template-oval_metadata) here

Reply:
will do